src/components/Navbar.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 
 const NavBar = () => (
```

Comment:
- [ ] Great job styling the app for the most part. :+1: Let's work on some minor details to match the design template. 
  - Kindly increase the vertical padding on the navbar to give it a bit more space. 🤝️
  - The `NavLink`s should be in uppercase to match the template. 🥸️
  - The title `Bookstore CMS` and the icon on the right should be aligned to the Book card below. 🙂️

**Your design**
![Screenshot from 2023-04-05 15-14-43](https://user-images.githubusercontent.com/17243325/230079698-d56c510b-4e97-4a95-9d15-681dacf277d3.png)

**Template**
![Screenshot from 2023-04-05 15-14-09](https://user-images.githubusercontent.com/17243325/230079688-f92472a0-f1ae-41e2-a891-b0db31d406a5.png)